config/initializers/inflections.rb

```diff
@@ -9,8 +9,8 @@
 #   inflect.singular /^(ox)en/i, '\1'
 #   inflect.irregular 'person', 'people'
 #   inflect.uncountable %w( fish sheep )
-  inflect.plural /^(embargo)$/i, '\1es'
-  inflect.singular /^(embargo)es/i, '\1'
+  inflect.plural(/^(alaveteli_pro_embargo|embargo)$/i, '\1es')
+  inflect.singular(/^(alaveteli_pro_embargo|embargo)es/i, '\1')
```

Comment:
```suggestion
  inflect.plural(/(embargo)$/i, '\1es')
  inflect.singular(/(embargo)es$/i, '\1')
```

Actually I think this does everything we need.

Reply:
Or even just:
```suggestion
  inflect.irregular 'embargo', 'embargoes'
```

Reply:
Ah yeah good call! Fixed in 56f27ac – will merge on green.

Reply:
Did you see my third suggestion: https://github.com/mysociety/alaveteli/pull/5615#discussion_r409661060

Reply:
I didn't – I think I prefer `plural` and `singular` because it doesn't feel as irregular as e.g. "person" vs "people" – its just that the `e` rule is slightly different. IDK – will keep as is for now I think 🤷 .

Reply:
Well... https://github.com/rails/rails/blob/b2eb1d1c55a59fee1e6c4cba7030d8ceb524267c/activesupport/lib/active_support/inflections.rb#L66